ggml/src/ggml-cuda.cu

```diff
@@ -1902,11 +1906,17 @@ static void ggml_cuda_mul_mat(ggml_backend_cuda_context & ctx, const ggml_tensor
             const int cc            = ggml_cuda_info().devices[id].cc;
             use_mul_mat_q           = use_mul_mat_q           && ggml_cuda_should_use_mmq(src0->type, cc, src1->ne[1]);
             any_gpus_with_slow_fp16 = any_gpus_with_slow_fp16 || !fast_fp16_available(cc);
+#ifdef GGML_USE_MUSA
+            use_mul_mat_vec_q       = false;
+#endif // GGML_USE_MUSA
         }
     } else {
         const int cc            = ggml_cuda_info().devices[ctx.device].cc;
         use_mul_mat_q           = use_mul_mat_q           && ggml_cuda_should_use_mmq(src0->type, cc, src1->ne[1]);
         any_gpus_with_slow_fp16 = any_gpus_with_slow_fp16 || !fast_fp16_available(cc);
+#ifdef GGML_USE_MUSA
+        use_mul_mat_vec_q       = false;
+#endif // GGML_USE_MUSA
```

Comment:
This is going to break types such as IQ quants that are not supported by dmmv.

Reply:
Yes, we are experiencing issues with enabling `use_mul_mat_vec_q`. For now, I have to explicitly disable it.

Reply:
After further investigation, it appears that the issue is linked to `vmm` enablement. To resolve the problem temporarily, I have disabled `vmm` and rolled back my previous modifications to the `use_mul_mat_vec_q` code.